### PR TITLE
Header markup and mobile view

### PR DESCRIPTION
**Pull request by SergeyShynkarenko**

### CODE REVIEW

File: app/scss/pages/_header.scss [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	height: 100vh;
+	width: auto;
+	background-repeat: no-repeat;
+	background-size: cover;
+	background-position: center;
+	z-index: 1;
+	&__top {
+		display: flex;
+		justify-content: space-between;
+		.hamburger {
```
Comment (IShinkarenko) [inline app/scss/pages/_header.scss]: hamburger move separately

File: app/scss/pages/_media.scss [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+}
+@media (max-width : 768px) {
+	
+}
+
+
+@media (max-width : 650px) {
+	.header {
+		&__top {
+			.hamburger {
```
Comment (IShinkarenko) [inline app/scss/pages/_media.scss]: Hamburger move separately

File: app/scss/pages/_header.scss [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+				width: 0;
+				left: auto;
+				right: 0;
+				bottom: -2px;
+				background-color: #fff;
+				height: 2px;	
+				transition: all .3s;
+			}
+			&:hover {
+				color: #fff;
```
Comment (IShinkarenko) [inline app/scss/pages/_header.scss]: remove color
remove fontweight
remove text decoration

File: app/index.html [hunk 4]
```diff
@@ -14,7 +14,55 @@
...
+			<div class="header__top">
+				<a class="hamburger hamburger--emphatic">
+					<span class="hamburger-box">
+						<span class="hamburger-inner"></span>
+					</span>
+				</a>
+				<a  class="header__logo" href="#">
+					<img src="images/dest/logo.svg" alt="Luxury Hotels">
+				</a>
+				<nav class="header__menu">
```
Comment (IShinkarenko) [inline app/index.html]: remove class="header__menu"

File: app/index.html [hunk 5]
```diff
@@ -14,7 +14,55 @@
...
+				<p class="header__text">Book your stay and enjoy Luxury<br>
+				redefined at the most affordable rates.</p>
+			</div>
+			<div class="calendar__button">
+				<a class="header__button button" href="#">BOOK NOW</a>
+			</div>
+			<div class="scroll__btn">
+				<p>Scroll</p>
+				<a href="#">
+					<img src="images/dest/arrow-scroll.svg" alt="">
```
Comment (IShinkarenko) [inline app/index.html]: fill alt

File: app/scss/_config.scss [hunk 6]
```diff
@@ -51,9 +52,45 @@ input, textarea {
...
+		z-index: -1;
+	}
+}
+.button {
+	display: inline-block;
+	text-align: center;
+	font-size: 17px;
+	font-weight: 400;
+	color: #fff;
+	padding: 20px 67px;
```
Comment (IShinkarenko) [inline app/scss/_config.scss]: padding: 15px 20px;

File: app/scss/pages/_header.scss [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+		line-height: 1.2;
+	}
+}
+.calendar__button {
+	text-align: center;
+	margin-bottom: 15px;
+	z-index: -1;
+	.header__button {
+		font-weight: 700;
+		padding: 15px 10px 15px 48px;
```
Comment (IShinkarenko) [inline app/scss/pages/_header.scss]: remove

File: app/scss/_config.scss [hunk 8]
```diff
@@ -51,9 +52,45 @@ input, textarea {
 	&:focus:required:invalid { border-color: red }
 	&:required:valid { border-color: green }
 }
+.container {
+	max-width: 1170px;
+	padding: 0 10px;
```
Comment (IShinkarenko) [inline app/scss/_config.scss]: 0 15px

File: app/scss/pages/_header.scss (hunk 1 above)
Comment (SergeyShynkarenko) [reply app/scss/pages/_header.scss:12]: done

File: app/scss/pages/_media.scss (hunk 2 above)
Comment (SergeyShynkarenko) [reply app/scss/pages/_media.scss]: done

File: app/scss/pages/_header.scss (hunk 1 above)
Comment (SergeyShynkarenko) [reply app/scss/pages/_header.scss]: done

File: app/scss/pages/_header.scss (hunk 3 above)
Comment (SergeyShynkarenko) [reply app/scss/pages/_header.scss]: done

File: app/index.html (hunk 4 above)
Comment (SergeyShynkarenko) [reply app/index.html]: done

File: app/index.html (hunk 5 above)
Comment (SergeyShynkarenko) [reply app/index.html]: done

File: app/scss/_config.scss (hunk 6 above)
Comment (SergeyShynkarenko) [reply app/scss/_config.scss]: done

File: app/scss/pages/_header.scss (hunk 7 above)
Comment (SergeyShynkarenko) [reply app/scss/pages/_header.scss]: done

File: app/scss/_config.scss (hunk 8 above)
Comment (SergeyShynkarenko) [reply app/scss/_config.scss]: done

**Review by SergeyShynkarenko — COMMENTED** [review]
done

**Pull request merged**